warehouse/ingest-core/src/main/java/datawave/ingest/data/config/ingest/BaseIngestHelper.java

```diff
@@ -796,7 +800,7 @@ protected Set<NormalizedContentInterface> normalize(NormalizedContentInterface n
                     values.add(normalize(normalizedContent, dataType));
                 }
                 if (log.isDebugEnabled()) {
-                    log.debug("added normalized field " + normalizedContent + " to values " + values);
+                    log.debug(ADDED_NORMALIZED_FIELD + normalizedContent + TO_VALUES + values);
```

Comment:
This `TO_VALUES` constant got me thinking that there's probably a cleaner way to address these. For one, this message is saying the same thing as the one above without the `TO_VALUES`. The only difference is we're logging the whole values set over and over here so I might go with the former.

Another thing we could do is add a method for `logNormalizedField(String value)` and that method could check the log level and log the message. Then we wouldn't have this block repeated over and over.

Reply:
Went for option 2. i think that seems cleaner and we can remove the 2 static strings.